active/Expander/Expander.md

```diff
@@ -0,0 +1,155 @@
+
+XAML
+~~~~
+<muxc:Expander x:Name="Expander2" Header="This is in the header"> 
+    <ToggleButton>This ToggleButton is in the Content</ToggleButton>
+</muxc:Expander>
+~~~~
+
+Updated pictures with final visuals to come.
+
+### Theme Resources
+
+You can modify the look of an Expander by specifying Xaml resources in your app.
+For more info, see the
+[lightweight styling guide](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/xaml-styles#lightweight-styling).
+
+| Name| Description | 
+| :---------- | :------- | 
+| ExpanderChevronMargin | Chevron margin thickness| 
+| ExpanderChevronGlyph | Chevron glyph| 
+| ExpanderChevronSize | Chevron size|
+| ExpanderPopinVerticalOffset | vertical offset for animation| 
+
+Some theme resources TBD.
+
+
+
+## Expander members
+
+### Properties
+
+| Name | Description | Default |
+| :---------- | :------- | :------- |
+| ExpandDirection | Sets the direction of expansion | Down = 0 |
+| IsExpanded | Whether or not control is expanded | False |
+
+### Events
+| Name | Description | 
+| :---------- | :------- | 
+| Expanding | Occurs when expanded |
+| Collapsed| Occurs when collapsed |
+
+
+
+# API Details
+
+~~~~
+enum ExpandDirection
+{
+    Down = 0,
+    Up = 1
+};
+
+runtimeclass ExpanderExpandingEventArgs
+{
+}
+
+runtimeclass ExpanderCollapsedEventArgs
+{
+}
+ 
+unsealed runtimeclass Expander : Windows.UI.Xaml.Controls.ContentControl
+{
+    Expander();
+
+    Object Header{ get; set; };
+    Windows.UI.Xaml.DataTemplate HeaderTemplate{ get; set; };
+    Windows.UI.Xaml.Controls.DataTemplateSelector HeaderTemplateSelector{ get; set; };
+
+    [MUX_PROPERTY_CHANGED_CALLBACK(TRUE)]
+    Boolean IsExpanded{ get; set; };
+
+    [MUX_DEFAULT_VALUE("winrt::ExpandDirection::Down")]
+    [MUX_PROPERTY_CHANGED_CALLBACK(TRUE)]
+    ExpandDirection ExpandDirection{ get; set; };
+
+    event Windows.Foundation.TypedEventHandler<Expander, ExpanderExpandingEventArgs> Expanding;
+    event Windows.Foundation.TypedEventHandler<Expander, ExpanderCollapsedEventArgs> Collapsed;
+
+    static Windows.UI.Xaml.DependencyProperty HeaderProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty HeaderTemplateProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty HeaderTemplateSelectorProperty{ get; };
+
+    static Windows.UI.Xaml.DependencyProperty IsExpandedProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty ExpandDirectionProperty{ get; };
+}
+~~~~
+
+# Inputs and Accessibility
+
+## UI Automation Patterns
+Expander will use a ExpandCollapsePattern. Expanding/Collapsing the expander will raise [RaisePropertyChanged](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.automation.peers.automationpeer.raisepropertychangedevent?view=winrt-19041) with the property changed being the [ExpandCollapseProperty](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.automation.expandcollapsepatternidentifiers.expandcollapsestateproperty?view=winrt-19041) property.
```

Comment:
(Low-pri) There was an ask in other documentation to remove the `en-us` tag so it lands on the clicker's preferred language.

Reply:
Thanks for the note! Making the changes